docs/rst/getting_started/entities.rst

```diff
@@ -140,25 +140,25 @@ Physical Entities
 
 Host
 ----
-*Host* make reference to the physical (or logical i.e. Docker) machine where one or some DDS *Processes*
+A *Host* makes reference to the physical (or logical i.e. Docker) machine where one or more DDS *Processes*
 are running.
 This entity is connected directly with the *Users* running in this *Host*.
 
 .. _user_entity:
 
 User
 ----
-*User* make reference to the different users that could run in a *Host*.
+A *User* makes reference to the different users that could run in a *Host*.
 
-This entity is connected directly with the *Host* it belongs, and the running *Processes* that are being executed
+This entity is connected directly with the *Host* it belongs to, and the running *Processes* that are being executed
 within this *User*.
 
 .. _process_entity:
 
 Process
 -------
-*Process* represents each process running an application using *Fast DDS*. Be aware that it is possible to
+A *Process* represents each process running an application using *Fast DDS*. Be aware that it is possible to
```

Comment:
Add this `A` to every definition or do not add it here.

Reply:
:+1: